grpc-utils/src/main/java/com/glegoux/grpc/GrpcServerRunner.java

```diff
@@ -0,0 +1,15 @@
+package com.glegoux.grpc;
+
+import java.io.IOException;
+
+public class GrpcServerRunner {
```

Comment:
general question: why do we need a runner now? Can't HelloServer just extend GrpcServer? or maybe I missed something

Reply:
I removed `GrpcServerRunner `. For the moment, I let `GrpcServer` as an interface not an abstract class. `HelloServer` re-uses the implementation without extending it. We can re-discuss on it later, but your suggestion could also be a possibility!